include/alpaka/acc/AccCpuOmp2Blocks.hpp

```diff
@@ -189,6 +189,13 @@ namespace alpaka
                 TKernelFnObj const& kernelFnObj,
                 TArgs&&... args)
             {
+                if(workDiv.m_blockThreadExtent.prod() != static_cast<TIdx>(1u))
```

Comment:
please remove the checks in the `operator()` no need to keep the old checks if we can not catch the exception.